interpretation/ww_head.py

```diff
@@ -860,7 +860,7 @@ def process(message, isGameMaster=False, isAdmin=False):
         # TODO
         return todo()
     if is_command(message, ['age'], True):
-        msg = "**USAGE:** This command is used to set your age. /n/n`" + prefix + "age<number>\`n\n**Example:** `!age 19`"
```

Comment:
Can you add replace this line again please?

Reply:
sure

Reply:
This one still has /n instead of \n

Reply:
Nevermind. It doesn't, but I don't see a replaced message?